packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -299,11 +300,21 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
       uint256 aStableTokenValueInGold = stableTokenSupply.mul(goldAmount).div(stableAmount);
       stableTokensValueInGold = stableTokensValueInGold.add(aStableTokenValueInGold);
     }
+    return
+      FixidityLib
+        .newFixed(reserveGoldBalance)
+        .divide(FixidityLib.newFixed(stableTokensValueInGold))
+        .unwrap();
```

Comment:
@asaj @trianglesphere In the code, we calculate the reserve ratio as cGLD-value over stabletokens-value. When designing the schedules and cutoffs we assume that the reserve ratio is total-reserve-value (including the BTC and ETH reserve funds) over stabletokens-value. Could we divide `reserveGoldBalance` by the the cGLD weight in `assetAllocationWeights` to approximate the reserve ratio as we define it when designing the schedules?

Reply:
That's actually what the tobin tax did in the first place (it did assume a 50% allocation into gold).
If it's hard enough to quickly compute ETH/BTC values in cGLD & the reserve stays close enough to it's target, using the target cGLD weight makes senses.